example/CMakeLists.txt

```diff
@@ -4,7 +4,7 @@ set (CMAKE_CXX_STANDARD 11)
 if (BUILD_READERS)
     add_executable (ZXingReader ZXingReader.cpp)
 
-    target_include_directories (ZXingReader PRIVATE ../thirdparty/stb)
+    target_link_libraries (ZXingReader stb::stb)
```

Comment:
Please combine that with the subsequent line like so `target_link_libraries (ZXingReader stb::stb ZXing::ZXing)`. The same for all other similar occurrences below.